airflow/www/templates/airflow/dag.html

```diff
@@ -110,33 +110,33 @@ <h4 class="pull-right" style="user-select: none;-moz-user-select: auto;">
     <div class="row">
       <div class="col-md-10">
         <ul class="nav nav-pills">
-          <li><a href="{{ url_for('Airflow.tree', dag_id=dag.dag_id, num_runs=num_runs_arg, root=root, base_date=base_date_arg) }}">
-            <span class="material-icons" aria-hidden="true">nature</span>
-            Tree
+          <li><a href="{{ url_for('Airflow.dag_grid', dag_id=dag.dag_id, num_runs=num_runs_arg, root=root, base_date=base_date_arg) }}">
+            <span class="material-icons" aria-hidden="true">grid_on</span>
+            Grid
           </a></li>
-          <li><a href="{{ url_for('Airflow.graph', dag_id=dag.dag_id, root=root, num_runs=num_runs_arg, base_date=base_date_arg, execution_date=execution_date_arg) }}">
+          <li><a href="{{ url_for('Airflow.dag_graph', dag_id=dag.dag_id, root=root, num_runs=num_runs_arg, base_date=base_date_arg, execution_date=execution_date_arg) }}">
             <span class="material-icons" aria-hidden="true">account_tree</span>
             Graph</a></li>
-          <li><a href="{{ url_for('Airflow.calendar', dag_id=dag.dag_id) }}">
+          <li><a href="{{ url_for('Airflow.dag_calendar', dag_id=dag.dag_id) }}">
```

Comment:
Instead of changing all these names, I’d just change the _old_ views to something like `legacy_calendar` and name the new view `calendar`. (But `tree` becomes `legacy_tree` and the new view `grid`).

Reply:
Done